app/assets/javascripts/debounce.js

```diff
@@ -0,0 +1,18 @@
+// Returns a function, that, as long as it continues to be invoked, will not
+// be triggered. The function will be called after it stops being called for
+// N milliseconds. If `immediate` is passed, trigger the function on the
+// leading edge, instead of the trailing.
+export function debounce(func, wait, immediate) {
```

Comment:
Ah, i haven't used this syntax of import/export before... perhaps we should rely on concatenation here? Primitive, i know...

Reply:
So wait, we're building with sprockets, i believe... old fashioned -- is this export/import syntax something that should work with that? Traditionally what I'd do is just make a `debounce` function and then concatenate it in by adding a ref to `debounce.js` in `application.js` -- not very nice but it works.

Reply:
Oh i see, this is ES6? https://stackoverflow.com/questions/950087/how-do-i-include-a-javascript-file-in-another-javascript-file#950146

Does this work locally?